index.d.ts

```diff
@@ -9,6 +9,7 @@ declare namespace FocusTrap {
     active?: boolean;
     paused?: boolean;
     focusTrapOptions?: FocusTrapOptions;
+    containerElements?: Array<HTMLElement | string>;
```

Comment:
Let's make this `Array<HTMLElement>` and call it done. Thanks for [the reminder](https://github.com/focus-trap/focus-trap-react/pull/179#discussion_r525302966) about the `prop-types` we also have, `containerElements: PropTypes.arrayOf(PropTypes.instanceOf(ElementType)),`, which will match this TypeScript declaration, and it should complain in dev builds if a string is passed-in. This will also sync with the README updates you've already made, `Array of Elements`. So everything about `containerElements` should be synced on `Element` objects only in the array.

Reply:
Oh, I missed this. Thank you @stefcameron! Updated now.